### DISCUSSION TITLE
Welcome to archinstall Discussions!

### DISCUSSION THREAD

**Discussion by Torxed** [archlinux/archinstall#73] [General]
##  GitHub Discussions

This is an optional place to discuss and/or get help with archinstall. 
Here you can:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members *(for anyone that doesn't use discord, irc or element/matrix)*.
  * Ask for general usage *(if you have a bug, it should be reported as a normal issue)*

It's always fun to hear how people use the product, so if you're up for it, comment below with some use cases :)

**Comment by hi-im-kent** [comment]
You helped me to install Arch, thanks mate

**Reply by Torxed** [reply]
Glad it helped! If you wished some other feature existed, drop a comment :)

**Comment by Col433** [comment]
Is there a way to set the locale to my country en_NZ.UTF-8 during install? It seems to use en_US.UTF-8 as defult. I can't understand why there were complaints about not having all the DE's available when all you need to do is to select xorg and install your favourite DE (eg xfce4)and login(eg sddm) under extra packages then run sudo systemctl enable sddm and reboot...done. I like this install script but keep it simple as possible.

**Reply by Col433** [reply]
This has now been solved by using archinstall --advanced before installing until it's added as a feature.

**Comment by Col433** [comment]
@Torxed do you have a timeframe for releasing version 2.2.0 or do we get more releases before then?

**Reply by Torxed** [reply]
@Col433 The locale will be introduced at some point probably. As to keeping it simple, this is the overall goal! Whenever we introduce more complexity - it makes it harder both for the users, maintainers and programmers.. So it's ultimately our goal to keep it as simple and light weight as possible.

Most likely 2.2.0 will be the next one, it might be delayed a tiny bit because life is hard and some of the features that we *need* is rather complicated to get right enough to be put into the wild :)

Unless there are some urgent bugs that needs fixing that can be achieved with very minimal work.

**Reply by Col433** [reply]
@Torxed Thanks Anton. This installer is looking great and working well. I have tried a few other installers and so far all make you install extra packages you don't want/need. This one is minimal, allows you to install only extra packages you want while being a quick and easy way to get Arch installed. I look forward to testing release 2.2.0 soon 😊

**Comment by Col433** [comment]
@Torxed Hi Anton. Great to see you and the team are making good progress with archinstall v2.3.0. Just out of interest do you test that archinstall works every month with the new iso release? I was going to try a new desktop on my spare computer and it installed ok then on rebooting it stops at device PARTUUID not found.... and drops me into rootfs. I know how to get out using mount /dev/sda2 new_root  exit.

**Reply by Torxed** [reply]
Hi and thanks! I personally do, and all the automated ISO builds here on GitHub that we do on every PR and commit uses the latest packages. The PARTUUID issue is a Grub issue I believe and is something we're working on fixing for encryption :)

**Reply by Col433** [reply]
@Torxed  I always use systemd and btrfs. Tried  Release v2.3.0.RC1 and for me it has issues too. I will keep testing new ISO builds and post again when archinstall is working on my system. I'm still using archinstall --advanced to get en_NZ UTF-8. I was hoping entering a locale option would be in v2.3.0 by default. Was reading the comments on Phoronix....some people think they know everything and quick to criticise but they don't come here to log and help solve problems....probably because their knowledge and ability would quickly get found out  😊

**Reply by Col433** [reply]
@Torxed  Release v2.3.0.RC2 installed on my system using my configuration all ok
